src/http_frontend/http_graph_executor_impl.cpp

```diff
@@ -0,0 +1,126 @@
+//*****************************************************************************
+// Copyright 2024 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include "http_graph_executor_impl.hpp"
+
+#include <string>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wall"
+#pragma GCC diagnostic ignored "-Wunused-but-set-variable"
+#include "tensorflow_serving/util/net_http/server/public/server_request_interface.h"
+#pragma GCC diagnostic pop
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#include "mediapipe/framework/calculator_graph.h"
+#include "mediapipe/framework/formats/image_frame.h"
+#include "mediapipe/framework/formats/image_frame_opencv.h"
+#pragma GCC diagnostic pop
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wall"
+#include "mediapipe/framework/formats/tensor.h"
+#include "mediapipe/framework/port/status.h"
+#pragma GCC diagnostic pop
+
+#include "../mediapipe_internal/mediapipe_utils.hpp"
+
+namespace ovms {
+
+Status deserializeInputSidePacketsFromFirstRequestImpl(
```

Comment:
It's intentional to leave it empty or it will be implemented later on?

Reply:
For now there is no known use case for side packets deserialization from requests, therefore it is enough implementation (returning ok and doing nothing)